2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_setup.py:_flush():76] Current SDK version is 0.15.3
2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_setup.py:_flush():76] Configure stats pid to 36657
2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_setup.py:_flush():76] Loading settings from /Users/lbokeria/.config/wandb/settings
2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_setup.py:_flush():76] Loading settings from /Users/lbokeria/Documents/projects/gbif-species-trainer-AMI-fork/model_training/wandb/settings
2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'model_training/04-train_model.py', 'program': '/Users/lbokeria/Documents/projects/gbif-species-trainer-AMI-fork/model_training/04-train_model.py'}
2023-05-30 14:25:13,222 INFO    MainThread:36657 [wandb_init.py:_log_setup():507] Logging user logs to /Users/lbokeria/Documents/projects/gbif-species-trainer-AMI-fork/model_training/wandb/run-20230530_142513-am0k0oyr/logs/debug.log
2023-05-30 14:25:13,223 INFO    MainThread:36657 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/lbokeria/Documents/projects/gbif-species-trainer-AMI-fork/model_training/wandb/run-20230530_142513-am0k0oyr/logs/debug-internal.log
2023-05-30 14:25:13,223 INFO    MainThread:36657 [wandb_init.py:init():547] calling init triggers
2023-05-30 14:25:13,223 INFO    MainThread:36657 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-05-30 14:25:13,223 INFO    MainThread:36657 [wandb_init.py:init():596] starting backend
2023-05-30 14:25:13,223 INFO    MainThread:36657 [wandb_init.py:init():600] setting up manager
2023-05-30 14:25:13,227 INFO    MainThread:36657 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-05-30 14:25:13,233 INFO    MainThread:36657 [wandb_init.py:init():606] backend started and connected
2023-05-30 14:25:13,240 INFO    MainThread:36657 [wandb_init.py:init():700] updated telemetry
2023-05-30 14:25:13,269 INFO    MainThread:36657 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2023-05-30 14:25:13,594 ERROR   MainThread:36657 [wandb_init.py:init():763] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-05-30 14:25:13,669 ERROR   MainThread:36657 [wandb_init.py:init():1168] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
Traceback (most recent call last):
  File "/Users/lbokeria/miniforge3/envs/gbif-species-trainer-AMI-fork/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1150, in init
    run = wi.init()
  File "/Users/lbokeria/miniforge3/envs/gbif-species-trainer-AMI-fork/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 769, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-05-30 14:25:14,678 WARNING MsgRouterThr:36657 [router.py:message_loop():77] message_loop has been closed
